scripts/seed.js

```diff
@@ -1 +1,34 @@
-console.log('hello');
\ No newline at end of file
+/* eslint-disable no-console */
+const fs = require('fs');
+const request = require('request');
+const path = require('path');
+
+const dir = 'data';
+
+// eslint-disable-next-line arrow-body-style
+const writeImgFile = (dirIdx, fileIdx) => {
+  return request('https://picsum.photos/200/300').on('error', (err1) => { console.log('error before pipe ', err1); }).pipe(fs.createWriteStream(path.join(__dirname, `${dir}`, `${dirIdx}`, `${fileIdx}.jpeg`))).on('close', ((err) => {
+    if (err) {
+      // throw err;
+      console.log('err ', err);
+    }
+    console.log('Image saved ', dirIdx, fileIdx);
+  }));
+};
+
+for (let i = 0; i < 100; i += 1) {
+  fs.mkdir(path.join(__dirname, `${dir}`, `${i}`), { recursive: true }, (err) => {
+    if (err) {
+      throw err;
+    } else {
+      console.log('Created new directory.');
+
```

Comment:
I'm curious why many setTimeouts here are necessary.  This seems like you could do this using one of the same in a fetch using whatever you want to use on the client side.  I do get the reasoning though.

Reply:
I was having trouble getting images from the api, the socket or port kept closing/dropping cause I think they couldn't handle me sending request for 600 images.